internal/openchoreo-api/services/component_service.go

```diff
@@ -444,23 +478,43 @@ func (s *ComponentService) GetComponentReleaseSchema(ctx context.Context, orgNam
 		}
 	}
 
+	// Build the wrapped schema properties
+	wrappedSchema := &extv1.JSONSchemaProps{
+		Type:       "object",
+		Properties: make(map[string]extv1.JSONSchemaProps),
+	}
+
+	// Only add componentTypeEnvOverrides if there are actual envOverrides
 	if baseParams != nil {
```

Comment:
Shall we use `envOverrides` like in `GetComponentSchema` function to be consistent?


```suggestion
	if envOverrides != nil {
```